LogLeadRecords.cls

```diff
@@ -24,7 +31,7 @@ public class LogLeadRecords{
 			}
 			else{
 				for(Database.Error err : sr.getErrors()){
-					Log__c record = new Log__c(Error__c = 'Corrupt Record');
+					Log__c record = new Log__c(Error__c = 'Record data not properly inserted..');

```

Comment:
Instead of user define message put system error message which is available in `Database.Error` object (`err`)